rdm/0003-persistent-identifiers.md

```diff
@@ -0,0 +1,105 @@
+- Start Date: 2019-09-30
+- RFC PR: [#3](https://github.com/inveniosoftware/rfcs/pull/3)
+- Authors: Lars Holm Nielsen
+
+# Persistent identifiers
+
+## Summary
+
+Overall design for management of persistent identifiers in Invenio RDM.
+
+## Motivation
+
+- As a user, I want to have a DOI for my upload, so that I I can cite my research.
+- As a admin, I want to manage multiple DOI prefixes, so that I I don’t need a repository installation per prefix.
```

Comment:
```suggestion
- As an admin, I want to manage multiple DOI prefixes, so that I don’t need a repository installation per prefix.
```

(The usage of `a` or `an` is a bit of an irregular mess -see what I did there? See https://english.stackexchange.com/questions/152/when-should-i-use-a-vs-an . Turns out I was probably wrong in thinking that "a RFC" was correct. Seem like "an RFC" works because the "R" there is pronounced like a vowel sound - "Arrrr" like a pirate. The "r" in request is not pronounced that way, however; so would require "an". And "user" is an exception to the rule: "a user" is correct. Go figure!).

More on point, when we say "manage multiple DOI prefixes", we mean accept DOI with different prefixes. But do we also mean, allow generation of DOI with different prefixes? Through a potential drop-down in the form for instance?

Reply:
We've got a use case for generating DOIs with different prefixes (some groups on campus use their own prefixes). How this could be implemented is flexible (different deposit forms, prefix connected to user accounts, drop-down in form).

Reply:
Thanks for the clarification @tmorrell